Microsoft.Toolkit.Uwp.SampleApp/SamplePages/AutoSelectBehavior/AutoSelectBehaviorPage.xaml

```diff
@@ -0,0 +1,21 @@
+﻿<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.AutoSelectBehaviorPage"
```

Comment:
You can actually remove this page and the backing cs file. Find the updated guidance in our [wiki](https://github.com/windows-toolkit/WindowsCommunityToolkit/wiki/Sample-Development).

Reply:
Done